runtime/lib/component/custodian.js

```diff
@@ -80,6 +87,12 @@ Custodian.prototype.onNetworkDisconnect = function onNetworkDisconnect () {
   if (wifi.getNumOfHistory() > 0) {
     logger.log('network switch, try to reconnect, waiting for user awake or button event')
     wifi.enableScanPassively()
+    if (this.promptTimer !== null) {
```

Comment:
Should this prompt only be setup on first network preparation? If so, I'd believe it's setup should be in `custodian.prepareNetwork`.

Reply:
Tx, moved to `custodian.prepareNetwork`.